pkg/runtimeproxy/server/docker/handler.go

```diff
@@ -125,8 +129,23 @@ func (d *RuntimeManagerDockerServer) HandleCreateContainer(ctx context.Context,
 			cfgBody.HostConfig.CpusetCpus = resp.Resources.CpusetCpus
 			cfgBody.HostConfig.CpusetMems = resp.Resources.CpusetMems
 			cfgBody.HostConfig.MemorySwap = resp.Resources.MemorySwapLimitInBytes
+			cfgBody.HostConfig.CgroupParent = resp.CgroupParent
 			podInfo.Resources = resp.Resources
 		}
+	} else {
+		resp := hookResp.(*v1alpha1.ContainerResourceHookResponse)
+		if resp.ContainerResources != nil {
+			cfgBody.HostConfig.CPUPeriod = resp.ContainerResources.CpuPeriod
+			cfgBody.HostConfig.CPUQuota = resp.ContainerResources.CpuQuota
+			cfgBody.HostConfig.CPUShares = resp.ContainerResources.CpuShares
+			cfgBody.HostConfig.Memory = resp.ContainerResources.MemoryLimitInBytes
+			cfgBody.HostConfig.OomScoreAdj = int(resp.ContainerResources.OomScoreAdj)
+			cfgBody.HostConfig.CpusetCpus = resp.ContainerResources.CpusetCpus
+			cfgBody.HostConfig.CpusetMems = resp.ContainerResources.CpusetMems
+			cfgBody.HostConfig.MemorySwap = resp.ContainerResources.MemorySwapLimitInBytes
+			cfgBody.HostConfig.CgroupParent = resp.CgroupParent
```

Comment:
Is it better to move  `cfgBody.HostConfig.CgroupParent = resp.CgroupParent` outside of this `if`, since hook plugin may only modify PodCgroupParent without modify container resources?